### ISSUE TITLE
Failed to build qt5-webkit on Mojave

### ISSUE THREAD

**Issue by william8th** [markwu/homebrew-personal#1]
Firstly, thank you for creating the script to install Zeal on Mac. Your work is highly appreciated! 👍 

I'm getting these errors trying to install qt5-webkit on Mojave and I think it might be related to https://github.com/OSGeo/homebrew-osgeo4mac/issues/481

```
==> Installing qt5-webkit from markwu/personal
==> Cloning https://github.com/qt/qtwebkit.git
Updating /Users/william.heng/Library/Caches/Homebrew/qt5-webkit--git
==> Checking out branch 5.212
Already on '5.212'
Your branch is up to date with 'origin/5.212'.
HEAD is now at 72cfbd7 Import WebKit commit 025cd3fc8d29620535309aa3e2ab7075aad36044
==> Patching
patching file Source/WTF/wtf/spi/darwin/XPCSPI.h
patching file Tools/qmake/projects/run_cmake.pro
==> /usr/local/opt/qt/bin/qmake ../WebKit.pro -config release -spec macx-clang
==> make
Last 15 lines from /Users/william.heng/Library/Logs/Homebrew/qt5-webkit/02.make:
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib: file: ../../lib/libWebCoreTestSupport.a(MockSourceBufferPrivate.cpp.o) has no symbols                                                         
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib: file: ../../lib/libWebCoreTestSupport.a(MockTracks.cpp.o) has no symbols                                                                      
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib: file: ../../lib/libWebCoreTestSupport.a(JSMockContentFilterSettings.cpp.o) has no symbols                                                     
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib ../../lib/libWebCoreTestSupport.a                                                                                                              
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib: file: ../../lib/libWebCoreTestSupport.a(PlatformSpeechSynthesizerMock.cpp.o) has no symbols                                                   
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib: file: ../../lib/libWebCoreTestSupport.a(MockBox.cpp.o) has no symbols                                                                         
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib: file: ../../lib/libWebCoreTestSupport.a(MockMediaPlayerMediaSource.cpp.o) has no symbols                                                      
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib: file: ../../lib/libWebCoreTestSupport.a(MockMediaSourcePrivate.cpp.o) has no symbols                                                          
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib: file: ../../lib/libWebCoreTestSupport.a(MockSourceBufferPrivate.cpp.o) has no symbols                                                         
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib: file: ../../lib/libWebCoreTestSupport.a(MockTracks.cpp.o) has no symbols                                                                      
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib: file: ../../lib/libWebCoreTestSupport.a(JSMockContentFilterSettings.cpp.o) has no symbols                                                     
[ 82%] Built target WebCoreTestSupport
make[2]: *** [all] Error 2
make[1]: *** [first] Error 2
make: *** [sub-Tools-qmake-projects-run_cmake-pro-make_first-ordered] Error 2
```

I saw that in the source code, you referred to the same repository so I was thinking if it's just the fact that the script has not been updated? I'm afraid I'm not a Ruby expert so if you could spare some time tweaking the script to support Mojave, that'd be great :)

**Comment by markwu** [comment]
I recompiled qt5-webkit on my mac with macOS 10.14.2, it seems no problem at all. Maybe missing some decencies. Can you try to install `libjpeg` and `libpng` and recompile it again?

`brew install libjpeg libpng`

**Comment by william8th** [comment]
Yeah! That seems to have worked. I've just tested this on a different laptop and it works.
What worked for me:
```
brew tap markwu/personal
brew install libjpeg libpng
brew install qt5-webkit
```


Weirdly on another laptop where I noticed this failure, I did:
```
brew tap markwu/personal
brew install qt
brew install qt5-webkit
```

The last step failed and I scrolled down through the instructions at https://github.com/zealdocs/zeal/issues/995#issuecomment-449609388 and did:
```
brew install libjpeg libpng
brew install qt5-webkit
```
which still failed thus my opening of this ticket. I think if you were to add `brew install libjpeg libpng` into your formula, it will work perfectly! Thank you once again for contributing your personal time to make this work 👍
